go-controller/pkg/crd/userdefinednetwork/v1/shared.go

```diff
@@ -26,6 +26,9 @@ const (
 
 // +kubebuilder:validation:XValidation:rule="has(self.subnets) && size(self.subnets) > 0", message="Subnets is required for Layer3 topology"
 // +kubebuilder:validation:XValidation:rule="!has(self.joinSubnets) || has(self.role) && self.role == 'Primary'", message="JoinSubnets is only supported for Primary network"
+// + ---
+// + TODO: Add the following validations when available (kube v1.31).
```

Comment:
we are already at 1.31 right? these 1.31 TODOs should be cleaned up now..

Reply:
Done

Reply:
I think we can clean up all TODOs now, as rebase has already happened. Maybe as a separate commit?

Reply:
Since its not related to MTU boundaries I think it should be done on different up PR.